infer_structcol/tests/test_inference.py

```diff
@@ -6,19 +6,21 @@
 from infer_structcol.main import Sample, Spectrum
```

Comment:
add ``*.DS_Store`` to the ``.gitignore`` file since we don't want git tracking all of these files - they will probably result in unnecessary merge conflicts somewhere down the line.